pkg/api/networkservice/mechanisms/sriov/constants.go

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2020 Cisco Systems, Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package sriov - parameters keys of SRIOV mechanism
+package sriov
+
+const (
+	// AvailablePortsKey - map key for 'Request.MechanismPreferences[0].Parameters' that contains a list of available ports
+	AvailablePortsKey = "availablePorts"
+
+	// EndpointPortKey - map key for 'Mechanism.Parameters' which contains selected port from endpoint
+	EndpointPortKey = "endpointPort"
+
+	// ConfigFileName - configuration file name
+	ConfigFileName = "configPorts.yml"
```

Comment:
I do not quite understand for what we need these params, can you attach a link to spec/issue?

Reply:
I realized that this is a wrong PR so I close it.